README.md

```diff
@@ -123,3 +127,17 @@ pytest --backend [speculos|ledgercomm|ledgerwallet] <tests/path>  # will run tes
 
 The tests of this repository are basically the same as this example, except
 the tests run on the three current firmwares (NanoS, NanoX and NanoS+).
+
+## Documentation
+
+You'll need the `doc` dependencies to generate the documentation:
```

Comment:
The complete documentation can be found `[here](https://ledgerhq.github.io/ragger/).`
If you want to generate it locally, you'll need the `doc` ...

Reply:
Yes, + I forgot to document the fact that `graphviz` is needed.